.github/workflows/smoke.yml

```diff
@@ -0,0 +1,71 @@
+name: Smoke tests
+env:
+  DOCKER_REGISTRY: "localhost:5000"
+  TAG: "latest"
+  DOCKER_ORG: enmasse
+  KUBERNETES_NAMESPACE: "enmasse-infra"
+  SYSTEMTEST_PROFILE: smoke
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@master      
+    
+    - name: setup-docker
+      uses: docker-practice/actions-setup-docker@0.0.1
+    
+    - name: Minikube setup with registry
+      uses: obabec/minikube-setup-action@v0.0.4-alpha
+      with:
+        k8s-version: 1.15.0
+    
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11.0.2
+
+    - name: Setup Ruby environment with ruby-build
+      uses: clupprich/ruby-build-action@v1.0.0
+  
+    - name: Install maven
+      run: sudo apt install maven
+  
+    - name: Install golang
+      run: |
+        sudo add-apt-repository ppa:longsleep/golang-backports
+        sudo apt-get update
+        sudo apt-get install golang-1.12
+
+    - name: Setup OLM
+      run: ./.travis/setup-olm.sh
+
+    - name: Install asciidoctor
+      run: gem install asciidoctor
+    
+    - name: Make images
+      run: make SKIP_TESTS=true
+      
+    - name: Build images
+      run: make docker_build
+      
+    - name: Tag images
+      run: make docker_tag
+      
+    - name: Push images
+      run: make -j 4 docker_tag docker_push
+      
+    - name: Generate templates
+      run: make templates
+
+    - name: Smoke tests
+      run: ./.travis/test.sh
+
+    - uses: actions/upload-artifact@v1
+      with:
+        name: test-logs
+        path: artifacts/
```

Comment:
should be /tmp/testlogs

Reply:
and also systemtests/target/surefire-reports should be archived

Reply:
Deploying artifacts is not fully implemented yet. I will try to implement this feature soon as possible.